[2021-05-26 10:12:18,262] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: Kuwait_DAG.GetCovidDataForKuwait 2021-05-25T10:11:03.490225+00:00 [queued]>
[2021-05-26 10:12:18,298] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: Kuwait_DAG.GetCovidDataForKuwait 2021-05-25T10:11:03.490225+00:00 [queued]>
[2021-05-26 10:12:18,299] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-26 10:12:18,301] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2021-05-26 10:12:18,302] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-26 10:12:18,327] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): GetCovidDataForKuwait> on 2021-05-25T10:11:03.490225+00:00
[2021-05-26 10:12:18,333] {standard_task_runner.py:52} INFO - Started process 80 to run task
[2021-05-26 10:12:18,338] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Kuwait_DAG', 'GetCovidDataForKuwait', '2021-05-25T10:11:03.490225+00:00', '--job-id', '16', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/covid_dag.py', '--cfg-path', '/tmp/tmpzymxnijg', '--error-file', '/tmp/tmpujcn58o4']
[2021-05-26 10:12:18,339] {standard_task_runner.py:77} INFO - Job 16: Subtask GetCovidDataForKuwait
[2021-05-26 10:12:18,455] {logging_mixin.py:104} INFO - Running <TaskInstance: Kuwait_DAG.GetCovidDataForKuwait 2021-05-25T10:11:03.490225+00:00 [running]> on host 431e058b07b3
[2021-05-26 10:12:18,523] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Kuwait_DAG
AIRFLOW_CTX_TASK_ID=GetCovidDataForKuwait
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T10:11:03.490225+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-25T10:11:03.490225+00:00
[2021-05-26 10:14:07,111] {python.py:118} INFO - Done. Returned value was: None
[2021-05-26 10:14:07,140] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=Kuwait_DAG, task_id=GetCovidDataForKuwait, execution_date=20210525T101103, start_date=20210526T101218, end_date=20210526T101407
[2021-05-26 10:14:07,225] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-26 10:14:07,249] {local_task_job.py:146} INFO - Task exited with return code 0
